VSOPushNugetPackages.cmd

```diff
@@ -0,0 +1,25 @@
+setlocal
```

Comment:
Yes, this is so we can publish the nuget packages to myget automatically during the signed build. I added basically a copy of PushNugetPackages.cmd, because the original uses TFS build variables that don't work with V2. I kept the old one for now so we don't break the existing builds. we can remove it when we switch over completely.